base/special/log.jl

```diff
@@ -409,3 +409,56 @@ function log1p(x::Float32)
     end
 end
 
+
+@inline function log_ext_kernel(x_hi::Float64, x_lo::Float64)
+    c10 = 0.116255524079935043668677
+    c9 = 0.103239680901072952701192
+    c8 = 0.117754809412463995466069
+    c7 = 0.13332981086846273921509
+    c6 = 0.153846227114512262845736
+    c5 = 0.181818180850050775676507
+    c4 = 0.222222222230083560345903
+    c3 = 0.285714285714249172087875
+    c2 = 0.400000000000000077715612
+    c1hi = 0.666666666666666629659233
+    c1lo = 3.80554962542412056336616e-17
+    estr =  evalpoly(x_hi, (c2, c3, c4, c5, c6, c7, c8, c9, c10))
+    res_hi = estr*x_hi
+    res_lo = fma(x_lo, estr, fma(estr, x_hi, -res_hi))
+    ans_hi = c1hi + res_hi
+    ans_lo = (c1hi - ans_hi) + res_hi + (res_lo + c1lo)
+    return ans_hi, ans_lo
+end
+
+# Log implimentation that returns 2 numbers which sum to give true value with about 68 bits of precision
```

Comment:
```suggestion
# Log implementation that returns 2 numbers which sum to give true value with about 68 bits of precision
```

I don't really understand "which sum to give true value". Also, is SleefPirates.jl the original source of this code or was it adapted from somewhere else originally?

Reply:
what I mean by this is that it outputs `yhi,ylo` such that `log(big(x))-yh-ylo <2^-68`, but the output is not normalized (`ylo !< eps(yhi)`). Basically for true double-double implementations you would typically guarantee that all the bits in `yhi` are correct, but I'm not doing that because it's not necessary for `pow`

Reply:
@chriselrod where did the SLEEFPirates code come from again?

Reply:
SLEEF: https://github.com/shibatch/sleef
@musm 's port: https://github.com/musm/SLEEF.jl
SLEEFPirates is a fork of the port.

Reply:
Probably?
The SLEEF.jl license mentions it's a port of SLEEF and includes a link, but has a different license and doesn't include a copy of SLEEF's.

I don't care about SLEEFPirates/its license, so probably just SLEEF.jl's is fine.